source/install/desktop-app-changelog.rst

```diff
@@ -3,12 +3,89 @@ Desktop Application Changelog
+~~~~~~~~~~~~~~~
+
+- 
+
+Highlights
+~~~~~~~~~~~~~~~
+
+- Added new branding for Desktop App v5.0.
+
+Improvements
+~~~~~~~~~~~~~~~
+
+MacOS
+^^^^^^
+- Made the window menu on macOS more consistent with system standards.
+
+All Platforms
+^^^^^^^^^^^^^
+- Added support for multiple languages to be used by the spellchecker.
+- Changed the server selection to use a dropdown instead of tabs.
+- Added a dark mode for settings and modals.
+- Added support for dragging and dropping of the server dropdown items.
+- Converted the tabs interface to support multiple configurable tabs based on the added server.
+- Removed the **Server Management** screen from **Settings**, and added Edit/Delete buttons to the new dropdown.
+- Updated some keyboard shortcuts and menu items to work with the new desktop app layout.
+- Updated loading screen visuals.
+- Added a checkbox to certificate error modal that allows users to permanently distrust a certificate.
+
+Architectural Changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Major version upgrade of Electron to v14.0. Electron is the underlying technology used to build the Desktop app.
+- Added rpm build option to the Electron builder.
+- Added Universal binaries for Mac users.
+- Migrated to Bootstrap v4 and refreshed the interface. Migrated to react-beautiful-dnd instead of react-smooth-dnd for a cleaner experience.
+
+Bug Fixes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Linux
+^^^^^^^^^^^^^
+- Fixed the tray icon size on Linux.
+- Fixed an issue where pressing ``Alt+<somekey>`` could cause the menu bar to disable and overlap the top bar on Linux.
+
+All Platforms
+^^^^^^^^^^^^^
+- Fixed an issue where resizing the app while in the System Console causes a white bar to appear at the top.
+- Fixed an issue where the right-click menu was missing from the ``jira connect`` modal.
+
+Known Issues
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- The new spellchecker connects to Google servers for downloading updated dictionaries.
+- Unread messages icon may be missing from the taskbar on Windows following 4.7.0 upgrade `MM-37807 <https://mattermost.atlassian.net/browse/MM-37807>`_.
+- The ``create_desktop_file.sh`` script is removed from the .tar.gz release. As a workaround, it can be downloaded from `GitHub here <https://github.com/mattermost/desktop/blob/master/src/assets/linux/create_desktop_file.sh>`_.
+- An error may occur when installing the MSI Installer on any Windows version.
+- Crashes might be be experienced in some Linux desktop clients. This is an upstream bug in the ``libnotifyapp`` library. A recommended workaround is to disable the system tray icon in the Desktop settings.
+- On some Linux distros, a sandbox setting is preventing apps from opening links in the browser (see https://github.com/electron/electron/issues/17972#issuecomment-486927073). While this is fixed for most installers, it is not on the tgz. In this case manual intervention is required via ``$ chmod 4755 <installpath>/chrome-sandbox``.
```

Comment:
Is this still an electron issue still @devinbinnie ?

Reply:
Yes I believe so.